src/cascadia/TerminalSettingsModel/WslDistroGenerator.cpp

```diff
@@ -188,6 +189,12 @@ static bool getWslNames(const wil::unique_hkey& wslRootKey,
             continue;
         }
 
+        const auto modernValue{ wil::reg::try_get_value<uint32_t>(distroKey.get(), RegKeyModern) };
+        if (modernValue == 1u)

```

Comment:
it's an `optional` - `nullopt` is `!=` 1 :)

Reply:
I'd prefer if this was `.value_or(0) != 0` because that would more accurately represent "truthy".